youtube_dl/extractor/drtv.py

```diff
@@ -35,16 +48,18 @@ def _real_extract(self, url):
                 'Video %s is not available' % video_id, expected=True)
 
         video_id = self._search_regex(
-            r'data-(?:material-identifier|episode-slug)="([^"]+)"',
+            (r'data-(?:material-identifier|episode-slug)="([^"]+)"',
+                r'data-resource="[^>"]+mu/programcard/expanded/([^"]+)"'),
             webpage, 'video id')
 
         programcard = self._download_json(
             'http://www.dr.dk/mu/programcard/expanded/%s' % video_id,
             video_id, 'Downloading video JSON')
         data = programcard['Data'][0]
 
-        title = data['Title']
-        description = data['Description']
+        title = self._og_search_title(webpage) or data['Title']
```

Comment:
Oh, you're absolutely right.

But the data['Title'] still uses ugly naming at least under nyheder. Should I make it prefer one or the other depending on which part of the site it's downloading from, or should I use string.replace to remove the suffix?